refinery/data_set_manager/utils.py

```diff
@@ -544,21 +544,19 @@ def generate_solr_params_for_user(params, user_uuid):
                 investigation=investigation
             )
         except Study.DoesNotExist:
-            continue
-            # It's not an error not to have data,
-            # but there's nothing more to do here.
+            logger.error('Expected at least one Study for %s', investigation)
+            raise
         except Study.MultipleObjectsReturned:
-            logger.error('Expected only one study for %s', investigation)
+            logger.error('Expected only one Study for %s', investigation)
```

Comment:
except (Study.DoesNotExist, Study.MultipleObjectsReturned) as e:

We tend to include e in the logger.